uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -64,13 +65,17 @@ define([
         }
       };
       vm.uiOptions = {
-        isAdmin: false,
         isChangeExpanded: false,
+        isManager: false, //when the dialog is opened by manager or admin
+        isOwner: false, //when the dialog is opened by the owner
         multipleDays: true,
+        contact: '',
         showDatePickerFrom: false,
         showDatePickerTo: false,
         userDateFormat: HR_settings.DATE_FORMAT,
+        selectedStatus: undefined,
         showBalance: false,
+        statusLabel: '',
```

Comment:
`vm.statusLabel` is required to be set before `vm.leaveRequestStatuses` gets filtered for manager and removes `waiting_approval` or whatever is the current status of leaverequest, using the indexBy idea in controller